db.py

```diff
@@ -5,7 +5,10 @@
 
 # Connect to DB and setup tables, if necessary
 DB_NAME = 'fbusers.db'
-DB_CONN = sqlite3.connect(DB_NAME, detect_types=sqlite3.PARSE_COLNAMES)
+DB_CONN = sqlite3.connect(DB_NAME,
+	detect_types=sqlite3.PARSE_COLNAMES,
+	check_same_thread=False #TODO this might be a terrible idea
```

Comment:
This worries me. If I'm not mistaken we're looking at potential data corruption.

What is keeping us from converting this (Lines 8 to 14) into a function which we call in each function below that needs to connect to the database? It seems that would be safer.

Reply:
This is going to take a rework of the db module to properly address. Let's handle this in a separate branch via issue #11